### PR TITLE
Adding Therminator2 and Therminator2ToHepMC parser build script

### PR DESCRIPTION
**Pull request by maciekniewielki**
The script tries to build from the official [Therminator2](https://github.com/alipwgmm/therminator) repo, which is waiting to be updated with the Therminator2ToHepMC parser

### CODE REVIEW

File: therminator2.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package: Therminator2
+version: "%(tag_basename)s%(defaults_upper)s"
```
Comment (ktf) [inline therminator2.sh]: Can you drop the `%(defaults_upper)s` part? We would like to move away from those for every recipe, and enforce them in the defaults themselves where it makes sense.

Comment (maciekniewielki) [reply therminator2.sh]: Done!

**Comment by maciekniewielki** [comment]
I would prefer to merge this pull request after the [Therminator2 parser request](https://github.com/alipwgmm/therminator/pull/1), because the building script expects the parser code to be there

**Comment by ktf** [comment]
Any news on this one?

**Comment by rbertens** [comment]
Hi @qgp, could you take a look at this  ?

**Comment by qgp** [comment]
Hi @maciekniewielki, @ktf, @rbertens,

sorry for the (huge) delay. I merged the changes to the therminator2 package in the alipwgmm request. I requested to add this package also to alisw (from where the official builds are done), see https://github.com/alisw/alidist/issues/923. 

Let me know if there is anything else missing from my side now?

Cheers,
   Jochen

**Comment by maciekniewielki** [comment]
From my side, everything should be ready.

@qgp Following the migration of therminator2 to the official repo, should we also change the link from which it should build to the new alisw repo?

**Comment by maciekniewielki** [comment]
@ktf Can we merge this pull request? I would like to start testing Therminator2 on the GRID.

**Pull request merged**